apps/prairielearn/src/pages/studentGradebook/studentGradebook.ts

```diff
@@ -66,14 +119,14 @@ router.get(
       StudentGradebookRowSchema,
     );
 
-    const stringifier = stringifyStream<StudentGradebookRow>({
+    const stringifier = stringifyStream<StudentGradebookRowRaw>({
       header: true,
       columns: ['Assessment', 'Set', 'Score'],
-      transform(row) {
+      transform(row: StudentGradebookRowRaw) {
```

Comment:
Nit, I usually like to let type inference do its thing:

```suggestion
      transform(row) {
```